digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/TextValueLinearAssetService.scala

```diff
@@ -127,8 +127,8 @@ class TextValueLinearAssetService(roadLinkServiceImpl: RoadLinkService, eventBus
     val euroAndExitPattern = "^[0-9|Ee][0-9|Bb]{0,2}"
     values.get match {
       case TextualValue(textValue) => textValue.split(",").forall(_.trim.matches(euroAndExitPattern)) match {
-          case true => print(textValue)
           case false => throw new AssetValueException(textValue)
+          case _ =>
```

Comment:
shouldn't this been taken care?

Reply:
The previous truthful case is bad, just printing for test purposes. This a correction to pass the validation

Reply:
At least send None, to avoid warnings.

Reply:
You can use `if` and `else` if it helps, something like: 

`values.get match {
      case TextualValue(textValue) =>
        if(!textValue.split(",").forall(_.trim.matches(euroAndExitPattern)))
          throw new AssetValueException("incorrect asset type")
      case _ => throw new AssetValueException("incorrect asset type")`